tests/python/relay/test_prelude.py

```diff
@@ -0,0 +1,12 @@
+import tvm
+from tvm import relay
+
+def load_prelude():
+    mod = relay.Module()
+    return relay.prelude.Prelude(mod)
+
+def test_list():
```

Comment:
Is this test supposed to do more?

Reply:
We don't seem to need this test file as long as we migrate all built n functions and ADTs to using this mechanism. `test_adt.py` needs to use Prelude so the tests there will cover this change as wel..

Reply:
I think we should incrementally move over. It is too much work to do in one pr.